docs/welcome/hello-world.md

```diff
@@ -99,7 +99,7 @@ Here we can see more of how Cpp2 makes it features work.
 **How: Simple, safe, and efficient by default.**
 
 - **Line 9: CTAD** just works, because it turns into ordinary C++ code which already supports CTAD.
-- **Lines 10-11: Automatic bounds checking** is added to `words[0]` and `words[1]` nonintrusively at the call site by default. Because it's nonintrusive, it works seamlessly with all existing container types that are `std::ssize`-aware, when you use them from safe Cpp2 code.
+- **Lines 10-11: Automatic bounds checking** is added to `words[0]` and `words[1]` nonintrusively at the call site by default. Because it's nonintrusive, it works seamlessly with all existing container types that are `std::size`-aware, when you use them from safe Cpp2 code.
```

Comment:
I waffled on `.size` and `.ssize`. Both are accurate... my understanding is that either both work or both don't work, depending on whether object has a `.size()` (or is a C array). I'm happy to switch to `std::size` here if it's less confusing, since the signedness isn't the point here and I want to minimize reader distraction in a Hello World section.

Reply:
> it looks like if the container doesn't have .size(), then the user can provide a std::size implementation, but that wouldn't automatically give it a std::ssize implementation, that would have to be provided as well.

I tested this with CE, and this is accurate.  

One thing that I didn't realize until now is that the "assert in bounds" function simply gets out of the way if it can't find working `size`, `ssize` and `begin` functions, and just does the operator[] call.  I wonder if it could eventually be made required by a `cppfront` option.

Reply:
Right, if the object has a reasonable `.size` that returns an integral type, then both `std::size` and `std::ssize` work. https://godbolt.org/z/KoxT4Tq83

That's an interesting detail about the user not having `.size`, and providing `std::size` but not `std::ssize`. That will not be bounds-checked. I decided to require that both be available, both because for every STL-like container they will be, and because if only one is available then there could be funkiness around signed/unsigned index values (which we try very hard to get exactly right including for literals). So I just required them both, and you're right we should probably say both are required (and that the are available by default for any object that has `.size()` returning an integer, including all STL-like containers).